app/controllers/admin/organisations_controller.rb

```diff
@@ -96,7 +96,7 @@ def destroy
 
   def get_layout
     design_system_actions = %w[confirm_destroy]
-    design_system_actions += %w[index show features] if preview_design_system?(next_release: false)
+    design_system_actions += %w[index show features people update] if preview_design_system?(next_release: false)
```

Comment:
i suspect `update` probably shouldn't be in here as it's linked to the edit view (which also isn't in here)

Reply:
Just double checked and this is the case 

<img width="785" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/1675eaf2-c493-4886-a7b2-c124493291b9">

Reply:
Nice, good catch ✅